api/index.py

```diff
@@ -5,24 +5,18 @@
 import json
 from init_db import *
 from urllib.request import urlopen
-import requests
-from lxml import etree
 
 # constants
 BOS_URLS = [
-		'https://bostest.api.blockgo.vip'
-		,'http://bos-test.eoshenzhen.io:8888'
-		,'https://bos-testnet.eosphere.io'
-		,'https://api.bostest.alohaeos.com'
-		,'https://boscore.eosrio.io'
+		'https://api.boscore.io'
+		,'https://bos.eoshenzhen.io:9443'
+		,'https://api-bos.eospacex.com'
 		]
 
-TALLY_API = "https://s3.amazonaws.com/bostest.referendum/referendum/tallies/latest.json"
-BOS_TEST_NET_EX = "https://bos-test.eosx.io/"
-BOS_NET_EX = "https://bos.eospark.com/"
-EOSX_VOTE_TOTAL_XPATH = u'//*[@id="__layout"]/div/div[2]/div[2]/div/div/div[5]/div/span/span/text()'
+TALLY_API = "https://s3.amazonaws.com/bos.referendum/referendum/tallies/latest.json"
+VOTE_TOTAL_API = "https://s3.amazonaws.com/bos.referendum/referendum/summaries/latest.json"
 
-BP_TOTAL_VOTES = 14620687
+BP_TOTAL_VOTES = 311193750652
```

Comment:
Make sure you are handling the /10000 correctly

311193750652 or 31119375.0652 ?

Reply:
I handle it in the next logic